website/donate/index.html

```diff
@@ -0,0 +1,81 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "_includes/base-resp.html" %}
+
+{% block page_title %}{{ _('Give to Thunderbird') }}{% endblock %}
+
+
+{% block header_content %}
+  <section class="text-left text-white flex flex-col items-start xl:w-full max-w-6xl mx-auto pl-8 pr-8">
+    <h1 class="font-4xl md:font-hero font-thin md:mt-10 mb-8">{{ self.page_title() }}</h1>
+  </section>
+  <nav role="directory" class="text-left text-white flex justify-center w-full max-w-6xl mx-auto bg-black-25 rounded mb-10">
+    <ul class="list-none p-0 flex flex-wrap justify-around flex-1 m-0">
+      <li class="ml-2 mr-2">
+        <a href="#donate" class="btn-link">{{ _('Support Thunderbird') }}</a>
+      </li>
+      <li class="ml-2 mr-2">
+        <a href="#faq" class="btn-link">{{ _('Frequently Asked Questions') }}</a>
+      </li>
+      <li class="ml-2 mr-2">
+        <a href="#ways-to-give" class="btn-link">{{ _('Other Ways to Give') }}</a>
+      </li>
+    </ul>
+  </nav>
+{% endblock %}
+
+{% block content %}
+  <section id="donate" class="pt-20 pb-20 flex justify-between items-center pr-8 pl-8">
+    <section class="flex flex-col md:flex-row justify-between mx-auto">
+      <aside class="flex flex-col flex-1 lg:max-w-4xl mr-8 ml-8 lg:ml-16 lg:mr-16 lg:pt-12">
+        <h3 class="header-section">
+          <span>{{ svg('heart') }}</span>
+          <aside class="flex flex-col">
+            {{ _('Support Thunderbird!')}}
+            <small class="font-normal font-md tracking-normal normal-case">
+              {{ _('Thunderbird is funded by users like you.') }}
+            </small>
+          </aside>
+        </h3>
+        <p class="font-lg tracking-wider leading-loose mb-6">
+          {{ _('Our mission is to provide an open source email app that is secure, private and free for users around the world. Your donations support this work. Will you give today?') }}
+        </p>
+        <p class="font-md tracking-wider leading-loose mb-6">
+          {{ _('Contributions are not tax-deductible as charitable donations.') }}
+        </p>
+      </aside>
+      <aside class="flex-1 max-w-xl text-center mx-auto w-full">
+          <a href="#XVFNMBAK" style="display: none"></a>
```

Comment:
nit: wrong indentation
```suggestion
        <a href="#XVFNMBAK" style="display: none"></a>
```